src/ui/components/InputRequest/InputRequest.test.tsx

```diff
@@ -101,44 +106,34 @@ describe("SetUserName component", () => {
   };
 
   test("It renders modal successfully", async () => {
-    const showSetUserName = true;
-    const setShowSetUserName = jest.fn();
     const { getByText } = render(
       <Provider store={storeMocked}>
-        <SetUserName
-          isOpen={showSetUserName}
-          setIsOpen={setShowSetUserName}
-        />
+        <InputRequest />
       </Provider>
     );
-    expect(getByText(EN_TRANSLATIONS.setusername.title)).toBeVisible();
-    expect(getByText(EN_TRANSLATIONS.setusername.button.confirm)).toBeVisible();
+    expect(getByText(EN_TRANSLATIONS.inputmodal.title.username)).toBeVisible();
+    expect(getByText(EN_TRANSLATIONS.inputmodal.button.confirm)).toBeVisible();
```

Comment:
Please change `inputmodal` to `inputrequest` so we are aligned.

Reply:
Done